index.html

```diff
@@ -0,0 +1,84 @@
+<html>
+<head>
+    <title>OfekTwitter</title>
+    </meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+    <link rel="icon" type="image/x-icon" href="images/favicon.ico">
+</head>
+<body >
+<div class= "topnav">
+    <img src="images/ofek.png">
+    <p>
+        OfekTwitter
+    </p>
+    <div class="topnav-right">
+        <a>Users</a>
+        <a>Sign up</a>
+        <a>Sign in</a>
+    </div>
+</div>
+<div class="twitter-body">
+    <div class="twitter-dashboard-left">
```

Comment:
Naming class convention :
naming the class: twitter-dashboard-left implays on the style that is being decided on in the css file. You should always name the class by its content. (for example if i decide in the future to put the followees on the left and this on the right the names of the class will be changed as well).

Reply:
change all classes names with "-right" and "-left"